packages/ckeditor5-list/src/listproperties/ui/listpropertiesview.js

```diff
@@ -178,19 +179,23 @@ export default class ListPropertiesView extends View {
 		super.render();
 
 		if ( this.stylesView ) {
-			for ( const styleButtonView of this.stylesView.children ) {
-				// Register the view as focusable.
-				this.focusables.add( styleButtonView );
-
-				// Register the view in the focus tracker.
-				this.focusTracker.add( styleButtonView.element );
-			}
+			this.focusables.add( this.stylesView );
+			this.focusTracker.add( this.stylesView.element );
 
 			// Register the collapsible toggle button to the focus system.
 			if ( this.startIndexFieldView || this.reversedSwitchButtonView ) {
 				this.focusables.add( this.children.last.buttonView );
 				this.focusTracker.add( this.children.last.buttonView.element );
 			}
+
+			for ( const item of this.stylesView.children ) {
+				this.stylesView.focusTracker.add( item.element );
+			}
+
+			this.stylesView.keystrokes = new KeystrokeHandler();
+			this.stylesView.keystrokes.listenTo( this.stylesView.element );
```

Comment:
These should be moved to `_createStylesView()` (per single responsibility principle). Let's not mix things when we have dedicated places to organize the code. These are internals of the `stylesView` and they should be initialized next to other internals.

Reply:
I moved initialisation of `stylesView.keystrokes` to `_createStylesView()`, but for the listener it would be too early (`stylesView.element` would be undefined) so I left the second line in place.

Reply:
Thanks for the tip with `render()`. Both lines are now in `_createStylesView()` 👍